.github/workflows/test.yaml

```diff
@@ -692,16 +723,11 @@ jobs:
       VERCEL_API_PROJECT_ID: ${{ secrets.VERCEL_API_PROJECT_ID }}
       VERCEL_API_ORG_ID: ${{ secrets.VERCEL_API_ORG_ID }}
```

Comment:
Do you think that we can de duplicate this as well?

Reply:
In theory yes, in practice the full platforms-serverless is up for a bigger rework anyway so I would tackle it via that.